Templates/run_single.jinja2

```diff
@@ -179,7 +179,7 @@
                 src="/static/relics/{{ key | urlencode }}.png"
                 onclick="neowText('{{ key }}',
                       {% if key in keys %}
-                      'Obtained on floor {{ keys[key] }}'
+                      'Obtained on floor {{ keys[key] }}<br>{{ path_as_list[keys[key] - 1].escaped_description() }}'
```

Comment:
here's the actual feature change. Added `path_as_list` because run.path is a generator and can't really index on it